resources/public/api/conf/1.0/scenarios/retrieveBusinessDetails.md

```diff
@@ -9,7 +9,44 @@
     <tbody>
         <tr>
             <td><p>N/A - DEFAULT</p></td>
-            <td><p>Simulate a successful response.</p></td>
+            <td><p>Simulate a successful response with a self-employment business.
```

Comment:
While this might render fine, the `<p>` tag should really end here - semantically, the HTML would make more sense like this:
```
<td>
    <p>Simulate a successful response with a self-employment business.</p>
    <p>Use the following businessId(s) for this scenario:</p>
    <ul>
        <li>XAIS123456789012</li>
    </ul>
</td>
```